web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -455,7 +448,9 @@ export function UnifiedResources(props: UnifiedResourcesProps) {
             {resourcesFetchAttempt.statusCode !== 400 &&
               resourcesFetchAttempt.statusCode !== 403 && (
                 <Box flex="0 0 auto" ml={2}>
-                  <ButtonLink onClick={onRetryClicked}>Retry</ButtonLink>
+                  <Button type="button" onClick={onRetryClicked}>
+                    Retry
+                  </Button>
```

Comment:
This button didn't look quite right. I changed it a little bit, but it still doesn't look the best. idk what to do here, as we can't use `fill` any other than `filled`, because otherwise it blends with the red background. Also I wish the language server would suggest me "custom-defined" props first, because now if I place my cursor inside `Button` and ask my editor what props are available, it suggests me a bunch of aria props. ;f

@gzdunek Do you remember if there was a reason `ButtonLink` was used here instead of `Button`? This link doesn't have a `href` and we add `onClick` on it, which makes `Button` more suitable for it.

| Before | After |
| --- | --- |
| <img width="986" alt="retry-before" src="https://github.com/gravitational/teleport/assets/27113/38dbd10b-0e7a-4dc3-93fa-4080df1597c2"> | <img width="986" alt="retry-after" src="https://github.com/gravitational/teleport/assets/27113/0ea16ddc-8946-480d-9d2e-2059d9e25051"> |

Reply:
>@gzdunek Do you remember if there was a reason ButtonLink was used here instead of Button? This link doesn't have a href and we add onClick on it, which makes Button more suitable for it.

Hmm it wasn't me who added that button 😅 but I agree, `Buttons` fits better here.

Reply:
Sorry, I just did a quick `git blame` and didn't dig further. 😏

Reply:
I'm going to leave it as is, since it's exactly how it looks now, and one of the next steps on my roadmap is to deal with the error banners, which will include modernizing their action buttons: https://www.figma.com/design/Gpjs9vjhzUKF1GDbeG9JGE/Application-Design-System?node-id=8890-5504&m=dev